buzzmobile/tools/route_mapper/googlemapskey.py.example

```diff
@@ -1,2 +0,0 @@
-# Has to have Google Maps Static API enabled in API Console
-googlemapskey=''
```

Comment:
This is the only world_model.png I've ever seen, so I don't know what I should revert it to. Plzhalp